.circleci/config.yml

```diff
@@ -0,0 +1,97 @@
+version: 2.1
+
+executors:
+  ubuntu-2004:
+    machine:
+      image: ubuntu-2004:202008-01
+
+workflows:
+  lambda-functions:
+    jobs:
+      - build:
+          context: login-sandbox
+
+jobs:
+  build:
+    executor: ubuntu-2004
+    steps:
+      - brew-install
+      - aws-cli-configure
+      - sam-cli-install
+      - checkout
+      - build-repo
+
+commands:
+  sam-cli-install:
+    steps:
+      - run:
+          name: "Install SAM CLI"
+          command: |
+            PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
+            brew tap aws/tap
+            brew install aws-sam-cli
+            # hack to fix broken brew install of aws-sam-cli
+            brew reinstall python@3.7
+            sam --version
+
+  brew-install:
+    steps:
+      - run:
+          name: "Install Brew"
+          command: |
+            sudo apt-get install build-essential
+            /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+            echo 'export PATH=/home/linuxbrew/.linuxbrew/bin:$PATH' >> /home/circleci/.bash_profile
+            eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
+            brew --version
+
+  aws-cli-install:
+    steps:
+      - run:
+          name: "Install AWS CLI"
+          command: |
+            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+            unzip awscliv2.zip
+            sudo ./aws/install
+
+  aws-cli-configure:
+    steps:
+      - run:
+          name: "Configure AWS CLI"
+          command: |
+            aws configure set default.aws_access_key_id $AWS_ACCESS_KEY_ID
+            aws configure set default.aws_secret_access_key $AWS_SECRET_ACCESS_KEY
+            aws configure set default.region $AWS_DEFAULT_REGION
+
+  build-repo:
+    steps:
+      - run:
+          name: "Validate SAM template"
+          command: |
+            PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
+            cd source
+            sam validate -t template.yaml --profile default --debug
+      - run:
+          name: "Build Functions"
+          command: | 
+            PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
+            cd source
+            sam build \
+              --template template.yaml \
+              --use-container
+      - run:
+          name: "Test Functions"
+          command: |
+            cd source
+      - run:
+          name: "Package Functions"
+          command: |
+            PATH=/home/linuxbrew/.linuxbrew/bin:$PATH
+            cd source
+            sam package \
+              --profile default \
+              --debug \
+              --s3-bucket $DEPLOY_S3_BUCKET \
+              --s3-prefix circleci/identity-infra-functions/$CIRCLE_BRANCH/packages \
```

Comment:
do we need this `circleci` here in the bucket path?

Reply:
Yes.  The bucket permissions given to the circle bot require this folder.